TempleEight/docs/reference/example-templefiles/airbnb.md

```diff
@@ -60,7 +60,7 @@ Review: service {
   tenant: Tenant;
   reservation: Reservation;
   stars: int(5, 0);
-  review: string;
+  comment: string;
```

Comment:
Could we match the one in the main repo? https://github.com/TempleEight/temple/blob/d30423eac2fd8f8ab276260575cd928547e059d9/examples/airbnb.temple#L73
```suggestion
  message: string;
```